tests/generation/test_utils.py

```diff
@@ -2637,6 +2637,21 @@ def test_beam_search_example_integration(self):
 
         self.assertListEqual(outputs, ["Wie alt bist du?"])
 
+    def test_beam_search_low_memory(self):
```

Comment:
Instead of adding an integration test, I'd like to request adding a mixin -- like [this one](https://github.com/huggingface/transformers/blob/a85ea4b19a47824a8e85d3304a698e2d5c8325ec/tests/generation/test_utils.py#L1470) for contrastive search with low memory.

That way, we can ensure it is fully compatible with all models :) Like in the test linked above, it is okay to skip some models if it is failing.

Reply:
Sure, sounds good, I will do it ;)

One question, should I add `@slow ` for this mixin test ?

Reply:
@Saibo-creator no need to add the `@slow` decorator -- we reserve that decorator to tests that use large models. In the mixing tests we are using small and randomly initialized models :)